FinSearchUnified/Bundle/SearchBundle/CriteriaRequestHandler/FindologicFacetCriteriaRequestHandler.php

```diff
@@ -80,7 +80,7 @@ private function isSearchPage(Request $request)
     {
         $params = $request->getParams();
 
-        return array_key_exists('sSearch', $params);
+        return array_key_exists('sSearch', $params) || $params['controller'] === 'search';
```

Comment:
Can we do like `$request->getControllerName() === 'search'` like we did in the method above `isCategoryListing`?